spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/system/OutputCapture.java

```diff
@@ -128,8 +129,11 @@ void reset() {
 
 	private String get(Predicate<Type> filter) {
 		Assert.state(!this.systemCaptures.isEmpty(),
-				"No system captures found. Check that you have used @RegisterExtension "
-						+ "or @ExtendWith and the fields are not private");
+				"No system captures found. When using JUnit 4, ensure that you have "
```

Comment:
We should probably just make this a general comment about making sure it's registered correctly. We don't really need to tie `OutputCapture` to JUnit at all now.

Reply:
Sounds reasonable to me.

Do you want me to update the PR, or will you just modify it yourself when merging?